src/storage/mod.rs

```diff
@@ -232,6 +232,14 @@ impl Command {
         }
     }
 
+    pub fn priority(&self) -> CommandPri {
+        self.get_context().get_priority()
+    }
+
+    pub fn need_flow_control(&self) -> bool {
+        !self.readonly() && self.priority() != CommandPri::High
```

Comment:
if the client has a bug, sending all command with high priority, how can we know this?